internal/executor/thread-mate/mentions.go

```diff
@@ -0,0 +1,16 @@
+package thread_mate
+
+import (
+	"fmt"
+	"strings"
+)
+
+func extractIDFromMention(in string) string {
+	in = strings.TrimPrefix(in, "<@")
+	in = strings.TrimSuffix(in, ">")
+	return in
+}
+
+func asMention(in string) string {
+	return fmt.Sprintf("<@%s>", in)
+}
```

Comment:
kinda, basically the executor plugin doesn't know what for which platform it was enabled, however in template I already use:
```
 Thanks for reaching out! Today, {{ .Assignee.ID | toMention }} will assist you in getting your Botkube up and running :botkube-intensifies:
```
so it will be just a manner of moving renderer into Botkube core. I will add a dedicated comment about it.